introduction.md

```diff
@@ -0,0 +1,144 @@
+
+- **Wallet**: an open-source software running with a bitcoind back-end used to keep track of the users' co-owned coins, transaction signatures, and to connect with a signing device and infrastructure servers.
+
+- **Signing Device**: a hardware module used to store private keys, public keys and wallet descriptors and to handle signing operations. As user verification is an important part of the Revault security model, such a device must allow the user to check the message integrity before signing it. 
+
+- **Coordinator server**: An always-online server used for efficient message routing among participants and other servers. Its primary use is for exchanging signatures and Spend TXs. 
+
+With these in place, stakeholders are able to delegate bitcoin to fund managers and manually broadcast Cancel, Emergency and Unvault-Emergency TXs. Fund managers will also be able to manually broadcast Cancel TXs to recover from mistakes. 
+
+Next, users who want additional features will require additional infrastructure components. Each stakeholder may control one or more watchtowers and/ or a co-signing server.
+
+- **Watchtower**: An always-online server that monitors the bitcoin network and chain, as well as polls the coordinator for Spend TXs to perform its responsibilities (automated unvault policy enforcement, automated spend policy enforcement, emergency deterrent). 
+
+- **Co-signing Server**: An always-online server that acts as a 'stupidly simple' anti-replay signing oracle: it will accept to sign any Spend TX, *but only once*. This is necessary to stop the managers from changing the Spend TX destination after the Unvault TX’s time-lock has expired. 
+
+Automatic unvault policy enforcement; requires a Watchtower. By default the Watchtower will broadcast a Cancel TX any time it detects an Unvault TX unless it adheres to a specific pre-configured unvault policy.
+
+Automatic spend policy enforcement; requires a Watchtower and Co-signing Server. By default the Watchtower will broadcast a Cancel TX any time it detects the Spend TX's parent Unvault TX unless it is made aware of the Spend TX in advance and that Spend TX adheres to a pre-configured spend policy. The co-signing server is a means to stop the managers from changing the Spend TX after the Unvault TX's time-lock has expired. 
+
+Automatic Emergency Deterrent: requires a *Watchtower* which monitors the bitcoin network and maintains multiple direct communication channels with the stakeholder. If the stakeholder triggers their panic button, a message is sent to the Watchtower which will broadcast all of its Emergency TXs and Unvault Emergency TXs. To further reduce the incentive of a theft attempt the broadcast of any Emergency TX associated with any vault triggers the broadcast of all Emergency TXs associated with all the remaining vaults.
```

Comment:
Bitcoin

Reply:
Panic button is only one example. One could have proof-of-life automatic emergency, or anything else, some automated.